src/target/tx/devo/common/msc2/usb_devo8.c

```diff
@@ -52,8 +52,10 @@ void MSC_Init() {
 
 void USB_Enable(unsigned use_interrupt)
 {
+#ifdef USB_ENABLE_PIN
     GPIO_setup_output(USB_ENABLE_PIN, OTYPE_PUSHPULL);
     GPIO_pin_set(USB_ENABLE_PIN);
+#endif
     //rcc_set_usbpre(RCC_CFGR_USBPRE_PLL_CLK_DIV1_5);
```

Comment:
replace with HAS_PIN(USB_ENABLE_PIN) {}
And then:
#define USB_ENABLE_PIN NULL_PIN